keyboards/divergetm2/readme.md

```diff
@@ -0,0 +1,21 @@
+# diverge tm2
+
+A 4x6x2 split ortholinear keyboard with 2u spacebars like the Levinson (similar to Let's Split). Made by [Unikeyboard](https://unikeyboard.io).
+
+Keyboard Maintainer: [IslandMan93](https://github.com/islandman93) and [xton](https://github.com/xton), I guess?
```

Comment:
Lines 5 and 6 both need to end with two spaces for the readme file to render as desired.

`, I guess?` should probably be removed from Line 5 as well. @drashna ?

Reply:
👍 
Yup.